docs/storybook-react/.npmrc

```diff
@@ -1 +0,0 @@
-ignore-scripts=true
```

Comment:
I believe this prevents postinstall scripts from running, including lavamoat, which we definitely want to ensure runs for this package, even if it's not released as a module.